### DISCUSSION TITLE
`{parents search:field="keyword"}` not working

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#2876] [Feature Requests]
How to reproduce: try searching in any field in `{parents}` tag.

Code example:
```
{exp:channel:entries channel="channel_two"}

<div>Title: {title}</div>
<ul>
{parents field="relationship" search:title="two"}
<li>Parents title: {parents:title}</li>
{/parents}
</ul>
  
{/exp:channel:entries}
```

**Comment by TomJaeger** [comment]
@shbchk Thanks for posting this.

 What version of EE and PHP are you currently running?

**Comment by shbchk** [comment]
@TomJaeger EE 7.2.4, PHP 7.4.21

**Comment by intoeetive** [comment]
I can confirm using `search` on `parents` tag does not influence the results.

Classifying this as feature request, as it's clearly not supported parameter: https://docs.expressionengine.com/latest/fieldtypes/relationships.html#parameters

**Comment by shbchk** [comment]
Thanks, Yuri! Happy New Year! 🎄

**Comment by pyrobob83** [comment]
+1 for this - running into a situation I need it and it's not there currently.

**Comment by shbchk** [comment]
This is how I'm currently doing this:

```
{parents field='relationship_field'}
  {if '{parents:my_field}' == '{segment_X}'}
   content
  {/if}
{/parents}
```
